README.md

```diff
@@ -35,16 +34,21 @@ The description of API:
 
 # How to start
 
-We [publish packages][packages] for major platforms on GitHub.
-
+We [publish packages][releases] for major platforms on GitHub.
 Each release comes with [release notes][release-notes] describing the
 installation and configuration instructions of the epoch node.
 
 Below is an overview of the installation process for different platforms,
 building the package from source, configuration and operation of the epoch
 node.
 
-[packages]: https://github.com/aeternity/epoch/releases
+[releases]: https://github.com/aeternity/epoch/releases
+[release-notes]: /docs/release-notes
+
+Please use the [latest published stable release][releases] rather than the [`master` branch][master].
```

Comment:
this can be linked to https://github.com/aeternity/epoch/releases/latest

Reply:
Ok, I will address in new PR.

Reply:
https://github.com/aeternity/epoch/pull/1868